### PR TITLE
Time will show

### PR DESCRIPTION
**Pull request by tormodv**
Solves issue #56 

All references to the calendar are using time:Instant with a selection of properties from the w3c time ontology.
Best to merge this after #57, overlap might occur.

### CODE REVIEW

**Review by kimviljanen — COMMENTED** [review]
When looking at the diff, I think in general the changes look correct. The concept of
hasTRS was something I didn’t yet understand completely (https://www.w3.org/TR/owl-time/#time:TRS) and how it fits CCDM.

Also I was thinking that is time:Instance (point in time without duration) always correct because e.g. a publication DAY might be considered to refer to the whole 24h long day (interval) and not just a 0 second point in time. So slightly unsure on this one, too.

**Comment by tormodv** [comment]
@kimviljanen TRS is the calendar system, i.e the Gregorian calendar that has been used in Rome since 1582, the most used today, but in other parts of the world, the change took a long time. Before that, the Julian calendar was in more common use, created 46 before Christ.

Publication day. This is a good one. We can say that a programme was published on 21-04-1968. Even if this is formally just a point in time with the part of the full description omitted, we often attribute an event to a day, and at the same time implicit think of a day as an interval with a 24H duration. 
Some broadcasters even operate with publication days that go from six in the morning to six the next morning attributing this to the date at noon.

File: ontology/EBUCore/ebucore.owl [hunk 1]
```diff
@@ -5029,7 +4923,6 @@ bbcconsepts:Organisation rdf:type owl:Class ;
 
 ###  http://www.bbc.co.uk/ontologies/coreconcepts/Person
 bbcconsepts:Person rdf:type owl:Class ;
```
Comment (alexandre-rouxel) [inline ontology/EBUCore/ebucore.owl:4925]: typo:  bbcconsepts => bbcconcepts

File: ontology/EBUCore/ebucore.owl [hunk 2]
```diff
@@ -10,6 +10,7 @@
 @prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
 @prefix skos: <http://www.w3.org/2004/02/skos/core#> .
 @prefix spin: <http://spinrdf.org/spin#> .
+@prefix time: <http://www.w3.org/2006/time#> .
 @prefix vann: <http://purl.org/vocab/vann/> .
 @prefix dcterms: <http://purl.org/dc/terms/> .
 @prefix bbcconsepts: <http://www.bbc.co.uk/ontologies/coreconcepts/> .
```
Comment (alexandre-rouxel) [inline ontology/EBUCore/ebucore.owl:16]: typo : bbcconsepts => bbcconcepts

Comment (JuergenGrupp) [reply ontology/EBUCore/ebucore.owl:16]: see #72

**Pull request merged**